journalbeat/README.md

```diff
@@ -0,0 +1,27 @@
+# journalbeat
+
+Journablbeat is a lightweight shipper for journald logs with Elasticsearch.
```

Comment:
Small typo 😄

Reply:
Journablbeat is perfectly cromulent! 😆 
Fixed